### ISSUE TITLE
Was KORDS rewritten from scratch?

### ISSUE THREAD

**Issue by gfarrell** [KCSU/kords#1]
Just out of curiosity -- I wrote [KORDS 4 (a decade ago, admittedly)](https://github.com/gfarrell/kords) and was wondering what happened to create an entirely new version?

**Comment by Genora51** [comment]
Yep, it was! I was a bit frustrated with the old site, and figured it was due an upgrade - so I just rewrote the entire thing from scratch essentially, with new design and using Laravel as the backend.

**Comment by gfarrell** [comment]
Great, glad someone picked it up. Are [Pnyx](https://github.com/gfarrell/pnyx) and Chiron still runnning?

**Comment by gfarrell** [comment]
Oh that's sad. Do you know if anyone is planning to resurrect?
